kivymd/textfields.py

```diff
@@ -53,7 +55,7 @@
             rgba: self.disabled_foreground_color if self.disabled else \
             (self.hint_text_color if not self.text and not self.focus else self.foreground_color)
 
-    font_name: 'Roboto'
+    font_name: DEFAULT_FONT_STYLE
```

Comment:
I would skip that one from the `DEFAULT_FONT_STYLE` because `'Roboto' != 'Body1'`

Reply:
Oops, my bad